test/chi0.jl

```diff
@@ -93,3 +94,105 @@ for temperature in (0, 0.03), spin_polarization in (:none, :collinear)
                   temperature=temperature, spin_polarization=spin_polarization)
     end
 end
+
+# ==================== Test χ0 with SCF extra bands ===========================
+
+function test_chi0_scf(testcase; symmetry=false, use_symmetry=false, temperature=0,
```

Comment:
The test above use fully converged states (cf eigen(Hk)) so we thought it would be interesting to turn that into a scfres-type interface. I agree that then the first test above can be removed, if you think that now it is useless.

Reply:
We also includeed chi0 test for metals to anticipate the future :)